lib/chocolatechip.js

```diff
@@ -202,18 +199,20 @@ global.ChocolateChip = {
   // MARK: Public
 
   eat(options) {
-    this._options = Object.assign({}, this._options, options);
+    if(defaultTranslations[options.language] == null){ options.language = 'en_GB' };
```

Comment:
Can be done immutable, also please use the AirBnB styleguide for if statements.

Immutable alternative:
```
const language = defaultTranslations[options.language] != null ? options.language : 'en_GB';
```